extensionsGallery-insider.json

```diff
@@ -408,13 +408,64 @@
                 ],
                 "statistics": [],
                 "flags": "preview"
-              }
+							},
+							{
+								"extensionId": "18",
+								"extensionName": "profiler",
+								"displayName": "SQL Server Profiler",
+								"shortDescription": "SQL Server Profiler for SQL Operations Studio",
+								"publisher": {
+									"displayName": "Microsoft",
+									"publisherId": "Microsoft",
+									"publisherName": "Microsoft"
+								},
+								"versions": [
+									{
+										"version": "0.1.0",
+										"lastUpdated": "5/22/2018",
+										"assetUri": "",
+										"fallbackAssetUri": "fallbackAssetUri",
+											"files": [
+											{
+												"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+												"source": "https://sqlopsextensions.blob.core.windows.net/march-extensions/profiler/agent-0.27.0.vs"
+											},
+											{
+												"assetType": "Microsoft.VisualStudio.Services.Links.Source",
+												"source": "https://github.com/Microsoft/sqlopsstudio"
+											},
+											{
+												"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+												"source": "https://raw.githubusercontent.com/Microsoft/vscode-mssql/master/images/sqlserver.png"
+											},
+											{
+												"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+												"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/extensions/profiler/README.md"
+											},
+											{
+												"assetType": "Microsoft.VisualStudio.Code.Manifest",
+												"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/extensions/profiler/package.json"
+											},
+											{
+												"assetType": "Microsoft.VisualStudio.Services.Content.License",
+												"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/LICENSE.txt"
+											}
+										],
+										"properties": [
+											{ "key": "Microsoft.VisualStudio.Code.ExtensionDependencies", "value": "Microsoft.mssql" },
+											{ "key": "Microsoft.VisualStudio.Code.Engine", "value": "*" }
```

Comment:
you can set this value to ">=0.30.2" so the extension can only be installed to sqlops after that version.